### ISSUE TITLE
Execution failed for task ':nodejs-download'

### ISSUE THREAD

**Issue by ThePrimedTNT** [Kotlin/kotlin-frontend-plugin#76]
```
* What went wrong:
Execution failed for task ':nodejs-download'.
> org/jetbrains/kotlin/preprocessor/PreprocessorKt
```

Anyone else having this issue?

**Comment by ThePrimedTNT** [comment]
Fixed by updating to 0.0.31

**Comment by wadejensen** [comment]
@ThePrimedTNT 
You just saved me hours. Legend.
